docs/tutorial/part-zero/index.md

```diff
@@ -10,47 +10,63 @@ Before you start building your first Gatsby site, you’ll need to familiarize y
-1. If not, download and install [Homebrew with the instructions](https://docs.brew.sh/Installation) for your operating system (Mac, Linux or Windows).
+1. If not, download and install [Homebrew with the instructions](https://docs.brew.sh/Installation).
 1. Once you've installed Homebrew, repeat step 2 to verify.
 
-### Mac Users: install Xcode Command Line Tools
+### Install Xcode Command Line Tools:
 
 1. Open your Terminal.
-1. On a Mac, install Xcode Command line tools by running `xcode-select --install`.
+1. Install Xcode Command line tools by running `xcode-select --install`.
    1. If that fails, download it [directly from the Apple site](https://developer.apple.com/download/more/), after signing-in with Apple developer account
 1. After being prompted to start the installation, you'll be prompted again to accept a software license for the tools to download.
 
-## ⌚ Install Node.js and npm
+## Install Node.js and npm
 
-Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer.
+Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer. npm comes bundled with Node.js so if you don't have npm, chances are that you don't have Node.js too.
 
 _Note: Gatsby's minimum supported Node.js version is Node 8, but feel free to use a more recent version._
 
 1. Open your Terminal.
-1. Run `brew update` to make sure you have the latest version of Homebrew.
-1. Run this command to install Node and npm in one go: `brew install node`
-
-Once you have followed the installation steps, make sure everything was installed properly:
-
-### Check your Node.js installation
-
-1.  Open up your terminal.
-2.  Run `node --version`. (If you’re new to the command line, “run `command`” means “type `node --version` in the command prompt, and hit the Enter key”. From here on, this is what we mean by “run `command`”).
-3.  Run `npm --version`.
-
-The output of each of those commands should be a version number. Your versions may not be the same as those shown below! If entering those commands doesn’t show you a version number, go back and make sure you have installed Node.js.
-
-![Check node and npm versions in terminal](01-node-npm-versions.png)
+1. Install Node.js for your appropriate operating system.
+
+   - On **MacOS** with homebrew installed, run `brew install node`. If you don't want to install it through homebrew, download the latest Node.js version from [the official Node.js website](https://nodejs.org/en/), double click on the downloaded file and go through the installation process.
+   - On **Windows**, download and install the latest Node.js version from [the official Node.js website](https://nodejs.org/en/)
+   - On **Linux**:
```

Comment:
Thinking we may be better off using an external resource here that we know will be kept up to date. https://nodejs.org/en/download/package-manager/

Reply:
It's true that we will not need to keep it updated, but doesn't having just a link, defeat the purpose of writing a guide? Like the instructions being more easy to find and not jumping from site to site to find instructions for every thing? 
I think changing a version number to keep it up to date (from node 11 to node 12) is not a big thing, and it will be kept updated by the huge community and contributors gatsby has.

If you agree I can add the link at the bottom, so if there are any breaking changes and the guide is not updated fast enough, people can find the solution.

Reply:
Absolutely, we don't want people to constantly have to go elsewhere. 
In this case though, the number of different options is a bit worrisome. There are so many to keep up to date and inaccurate information can be incredibly problematic for something labeled as step 0. 
We have a link in the Mac and Windows instructions, this feels like a continuation of that.
But I'm open to other opinions!

Reply:
Yes I agree with that.
Well, considering Linux users have probably used the command line before and maybe installed a couple of packages, we could just link the official instructions? 
Or maybe add instructions for Ubuntu, and link this guide underneath in case they are on a different distro?

Reply:
I think the official instructions are the way to go. Are you good to make that change? I'd love to get this merged in soon! It's going to be a great addition.

Reply:
Yes, sorry for being so late, I will surely change it in the next days max when I get home. 
Thanks for all the suggestions!